requirements.txt

```diff
@@ -1,3 +1,4 @@
 argparse==1.2.1
 pyee==0.1.0
 wsgiref==0.1.2
+pdoc==0.3.1
```

Comment:
this does not belong in requirements.txt, as it's only a requirement for generating the docs, not for using the library.